scripts/scr_weapon/scr_weapon.gml

```diff
@@ -1681,6 +1542,166 @@ global.weapons = {
 		"arp": 0,
 		"tags": ["vehicle", "turret", "volkite", "ancient"]
 	},
+// Hireling weapons
+	// Admech
+	"Hellgun": {
+		"abbreviation": "HllGun",
+		"attack": {
+			"standard": 30,
+			"master_crafted": 35,
+			"artifact": 40
+		},
+		"description": "Known as a Hellgun or Hot-shot Lasgun, these high-energy weapons are more potent and destructive than common Lasguns, drawing more power for a more penetrative shot, but also more complex, requiring reinforced barrels, thermal-cooling cells and gyro-stabilized power packs.",
+		"ammo": 20,
+		"range": 12,
+		"spli": 3,
+		"arp": 0,
+		"tags": ["las"],
+		"melee_hands": 0,
+		"ranged_hands": 1,
+	},
+	"Laspistol": {
+		"attack": {
+			"standard": 20,
+			"master_crafted": 30,
+			"artifact": 40
+		},
+		"description": "The Laspistol is the pistol version of the Lasgun and like that weapon fires a coherent beam of energetic photons that can burn through most materials. The Laspistol is powered by a miniature power pack that is usually placed within the grip.",
+		"abbreviation": "Lpstl",
+		"melee_hands": 0,
+		"ranged_hands": 0.25,
+		"ammo": 30,
+		"range": 3.1,
+		"spli": 0,
+		"arp": 0,
+		"tags": ["pistol", "las"],
+	},
+	// Other imperials
+	"Light Bolter": {
+		"abbreviation": "LiBltr",
+		"attack": {
+			"standard": 40,
+			"master_crafted": 45,
+			"artifact": 50
+		},
+		"description": "A smaller variant of Bolter, intended to be useable by unaugmented humans.",
+		"melee_hands": 0,
+		"ranged_hands": 1,
+		"ammo": 16,
+		"range": 12,
+		"spli": 3,
+		"arp": 0,
+		"tags": ["bolt"]
+	},
+	"Sarissa": {
+		"abbreviation": "Saris",
+		"attack": {
+			"standard": 40,
+			"master_crafted": 45,
+			"artifact": 50
+		},
+		"description": "A vicious combat attachment that is attached to Bolters, in order to allow them to be used in melee combat.",
+		"melee_hands": 0,
+		"ranged_hands": 0,
+		"ammo": 0,
+		"range": 1,
+		"spli": 2,
+		"arp": 0,
+		"tags": ["sword"],
+	},
```

Comment:
Nerfed Sarissa to be on pair with astartes Combat Knife, as this makes the most sense.
It's also mentioned that this is used as an axe and not sword.
```suggestion
	"Sarissa": {
		"abbreviation": "Saris",
		"attack": {
			"standard": 25,
			"master_crafted": 30,
			"artifact": 35
		},
		"description": "A vicious combat attachment that is attached to Bolters, in order to allow them to be used in melee combat.",
		"melee_hands": 0,
		"ranged_hands": 0,
		"ammo": 0,
		"range": 1,
		"spli": 1,
		"arp": 0,
		"tags": ["axe"],
	},
```